test/higher_order_ops/test_print.py

```diff
@@ -0,0 +1,45 @@
+# Owner(s): ["module: higher order operators"]
+import io
+from unittest.mock import patch
+
+import torch
+from torch._dynamo.utils import counters
+from torch.testing._internal.common_utils import run_tests, TestCase
+
+
+class TestHopPrint(TestCase):
+    def test_base_print(self):
+        def f(x):
+            x = x + x
+            torch._higher_order_ops.print("moo")
```

Comment:
try replacing this with `torch.ops.higher_order.print` -- I believe you will run into an issue since below you did `super().__init__("hop_print")`